Bindings/bindings.json

```diff
@@ -6,473 +6,488 @@
+          "value": "string",
+          "defaultValue": "MyCollection",
+          "required": true,
+          "label": "To which collection should the data be written?",
+          "help": "Please enter the name of the collection within your database."
+        },
+        {
+          "name": "createIfNotExists",
+          "value": "boolean",
+          "defaultValue": false,
+          "required": true,
+          "label": "Would you like the DocumentDB database and collection to be created for you?",
+          "help": "When true, your database and collection will be created automatically."
+        },
+        {
+          "name": "connection",
+          "value": "string",
+          "resource": "DocumentDB",
+          "required": true,
+          "label": "Which DocumentDB account should be used?",
+          "help": "[variables('appSettingsHelp')]"
+        }
+      ]
+    },
+    {
+      "type": "easyTable",
+      "displayName": "Azure Mobile Easy Table Record",
+      "direction": "out",
+      "defaultParameterName": "record",
+      "parameterNamePrompt": "Name the record for use in your code",
+      "settings": [
+        {
+          "name": "tableName",
+          "value": "string",
+          "defaultValue": "MyTable",
+          "required": false,
+          "label": "To which table should the data be sent?",
+          "help": "Please enter the name of the table within your Mobile App. If you are using a strongly-typed parameter in C# this property is not required."
+        }
+      ]
+    },
+    {
+      "type": "easyTable",
+      "displayName": "Azure Mobile Easy Table Record",
+      "direction": "in",
+      "defaultParameterName": "record",
+      "parameterNamePrompt": "Name the record for use in your code",
+      "settings": [
+        {
+          "name": "tableName",
+          "value": "string",
+          "defaultValue": "MyTable",
+          "required": false,
+          "label": "To which table should the data be sent?",
+          "help": "Please enter the name of the table within your Mobile App. If you are using a strongly-typed parameter in C# this property is not required."
+        },
+        {
+          "name": "id",
+          "value": "string",
+          "defaultValue": "{id}",
+          "required": true,
+          "label": "What is the Id for the record?",
+          "help": "Please enter the id for the record to retrieve."
+        }
+      ]
+    },
+    {
+      "type": "notificationHub",
+      "displayName": "Azure NotificationHub",
+      "direction": "out",      
+      "settings": [
+        {
+          "name": "tagExpression",
+          "value": "string",
+          "defaultValue": "tag1||tag2",
+          "required": false,
+          "label": "Optional.Tag expression for sending push notification",
+          "help": "Tag expressions enable you to target specific sets of devices/registrations, when sending a push notification through Notification Hubs."
         }
```

Comment:
we need to have a setting like this https://github.com/pragnagopa/azure-webjobs-sdk-templates/blob/nhOut/Bindings/bindings.json#L426-L431
to select Notification hub